client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -83,17 +84,22 @@ class RegistrantExtraInfoFrForm extends React.PureComponent {
 		//    fields so we can keep them together in one place
 		defaultRegistrantType = this.props.contactDetails.organization ? 'organization' : 'individual';
 
-		this.props.updateContactDetailsCache( {
+		const payload = {
 			extra: {
 				fr: { registrantType: defaultRegistrantType },
 			},
-		} );
+		};
+
+		this.props.updateContactDetailsCache( payload );
+		this.props?.onContactDetailsChange( payload );
 	}
 
 	updateContactDetails( field, value ) {
 		const sanitizedValue = this.sanitizeField( field, value );
 		debug( 'Setting ' + field + ' to ' + value );
-		this.props.updateContactDetailsCache( set( {}, field, sanitizedValue ) );
+		const payload = set( {}, field, sanitizedValue );
+		this.props.updateContactDetailsCache( payload );
```

Comment:
Interestingly, `updateContactDetailsCache` is not marked as required in the proptypes, and it has no default, so this (and every instance of that function in this class) should be `this.props.updateContactDetailsCache?.(payload)`. Alternatively, the proptype should be changed to make it required since that's the current behavior.

I know that's not part of your PR but it's a small fix for an existing bug and I'd say it's worthwhile.